platform-http-proxy/src/main/java/org/codice/ddf/platform/http/proxy/HttpProxy.java

```diff
@@ -117,4 +130,55 @@ public void stopProxy() throws Exception {
     public void setHostname(String hostname) {
         this.hostname = hostname;
     }
+
+    public static class PolicyRemoveBean {
+
+        private final String httpPort;
+
+        private final String httpsPort;
+
+        private final String host;
+
+        private final String cxfContext;
+
+        public PolicyRemoveBean(String httpPort, String httpsPort, String host, String cxfContext) {
+            this.httpPort = httpPort;
+            this.httpsPort = httpsPort;
+            this.host = host;
+            this.cxfContext = cxfContext;
+        }
+
+        public void rewrite(Exchange exchange) throws IOException {
+            try {
+                Message httpMessage = exchange.getIn();
+                if (httpMessage != null) {
+                    Object body = httpMessage.getBody();
```

Comment:
It does not look like you use body outside of the check in the next line.  You can combine them.